Lesson_27/Homework_27.py

```diff
@@ -0,0 +1,63 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+import time
+
+
+class NovaPoshtaTracking:
+    def __init__(self, driver_path):
+
+        self.driver = webdriver.Chrome(executable_path=driver_path)
```

Comment:
For now is no need to download driver and provide it`s path. You can use DriverManager for automatic downloading.